app/views/analytics/_customer_analytics.haml

```diff
@@ -1,38 +1,17 @@
-- if feature_enabled?(:customer_universal_analytics)
```

Comment:
Is the flag still used somewhere? If not, it should be removed from the Flag Store.